pkg/ocm/versions.go

```diff
@@ -29,7 +29,7 @@ const (
 	DefaultChannelGroup   = "stable"
 	LowestSTSSupport      = "4.7.11"
 	LowestSTSMinor        = "4.7"
-	LowestHostedCPSupport = "4.11.6-candidate"
+	LowestHostedCPSupport = "4.11.6"
```

Comment:
i dont think nightly have patch. For eg. 4.11.0-0.nightly-2022-10-17-040259

Reply:
it will still work, the code looks for `-0.nightly` as prefix of the Prerelease field (whatever is after the semantic versioning):
https://github.com/openshift/rosa/pull/895/files#diff-83f815beabf111da90b662b338ca5594199acfab18880dd9ed5c2ee4dc100d12R115

Reply:
can we check the channleGroup of the version instead of parsing the version format

Reply:
yeah, much better. thanks for the suggestion!. Code updated :smile: